projects/docs/docs/manifests/project.md

```diff
@@ -231,7 +231,20 @@ It represents a list of glob patterns that refer to files:
 
 | Property | Description                | Type            | Required | Default |
 | -------- | -------------------------- | --------------- | -------- | ------- |
-| `globs`  | Glob pattern to the files. | [`[Path]`](#path) | Yes      |         |
+| `globs`  | Glob pattern to the files. | [`[FileListGlob]`](#file-list-glob) | Yes      |         |
```

Comment:
Could you adjust the header "columns" to be of the same size of the row?

Reply:
fixed